beekeeper-core/src/test/java/com/expediagroup/beekeeper/core/repository/HousekeepingMetadataRepositoryTest.java

```diff
@@ -57,8 +57,13 @@
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)
 public class HousekeepingMetadataRepositoryTest {
```

Comment:
This is in the [ExpiredMetadataHousekeepingGeneratorTest](https://github.com/ExpediaGroup/beekeeper/blob/3461c4ad35e5953f00ea9f2f34fe886b72ac447c/beekeeper-scheduler-apiary/src/test/java/com/expediagroup/beekeeper/scheduler/apiary/generator/ExpiredHousekeepingMetadataGeneratorTest.java) because that's where all the different cases are handled.